docs/src/examples/Aerosol_activation_example.md

```diff
@@ -0,0 +1,51 @@
+# Aerosol Activation Example
+
+
+### Overview
+
+This example is based on AerosolActivation module which is a part of the
+  [CloudMicrophysics.jl](https://github.com/CliMA/CloudMicrophysics.jl) package.
+The AerosolActivation module computes the total number and mass
+  of aerosol particles that get activated and become cloud droplets,
+  given the atmospheric conditions and
+  the initial aerosol size distribution and properties.
+See the AerosolActivation module
+  [docs](https://clima.github.io/CloudMicrophysics.jl/dev/Aerosol_activation/)
+  for derivation and description of all input parameters.
+
+In this example we use the ensemble Kalman methods to learn
+  two parameters that describe the chemical composition of aerosol particles
+  based on the observed total number and mass of activated particles.
+The AerosolActivation model is used here in a "perfect model" setting,
+  meaning that the observations are generated by the same module
+  we are calibrating.
+Also, the setup of the example is very simple
+  and meant to be an entry level tutorial.
+In future aplications the artificially generated observations
+  can be swapped with actual measurements,
+  and the calibration setup should be expanded to include more combinations
+  of atmospheric conditions and initial aerosol size distributions.
+
```

Comment:
Remove ~~Also, the setup of the example is very simple
  and meant to be an entry level tutorial. In future aplications the artificially generated observations
  can be swapped with actual measurements,
  and the calibration setup should be expanded to include more combinations
  of atmospheric conditions and initial aerosol size distributions.~~

"swapping with actual measurements" requires dealing with model form error, which is likely non-trivial. I would stick to defining what is done here.